docs/src/tutorials/Getting started/an_introduction_to_julia.jl

```diff
@@ -0,0 +1,400 @@
+# # Getting started with Julia
+
+# **Originally Contributed by**: Juan Pablo Vielma
+
+# Since JuMP is embedded in Julia, knowing some basic Julia is important
+# for learning JuMP. This tutorial is designed to provide a minimalist
+# crash course in the basics of Julia. You can find resources that provide
+# a more comprehensive introduction to Julia [here](https://julialang.org/learning/).
+
+# ## How to print
+
+# In Julia, we usually use println() to print
+
+println("Hello, World!")
+
+# ## Basic data types
+
+# Integers
+
+typeof(1 + -2)
+
+# Floating point numbers
+
+typeof(1.2 - 2.3)
+
+# There are also some cool things like an irrational representation of π. To
+# make π (and most other greek letters), type `\pi` and then press `[TAB]`.
+
+π
+
+#-
+
+typeof(π)
+
+# Julia has native support for complex numbers
+
+typeof(2 + 3im)
+
+# Double quotes are used for strings
+
+typeof("This is Julia")
+
+# Unicode is fine in strings
+
+typeof("π is about 3.1415")
+
+# Julia symbols provide a way to make human readable unique identifiers.
+
+:my_id
+typeof(:my_id)
+
+# ## Arithmetic and Equality Testing
+
+# Julia is great for math
+
+1 + 1
+
+# Even math involving complex numbers
+
+(2 + 1im) * (1 - 2im)
+
+# We can also write things like the following using √ (\sqrt)
+
+sin(2π/3) == √3/2
+
+# Wait. What???
+
+sin(2π/3) - √3/2
+
+# Let's try again using ≈ (\approx).
+
+sin(2π/3) ≈ √3/2
+
+# Note that this time we used ≈ instead of ==. That is because computers don't
+# use real numbers. They use a discrete representation called floating point. If
+# you aren't careful, this can throw up all manner of issues. For example:
+
+1 + 1e-16 == 1
+
+# It even turns out that floating point numbers aren't associative!
+
+(1 + 1e-16) - 1e-16 == 1 + (1e-16 - 1e-16)
+
+# ## Vectors, matrices and arrays
+
+# Similar to Matlab, Julia has native support for vectors, matrices and tensors;
+# all of which are represented by arrays of different dimensions. Vectors are
+# constructed by comma-separated elements surrounded by square brackets:
+
+b = [5, 6]
+
+# Matrices can by constructed with spaces separating the columns, and semicolons
+# separating the rows:
+
+A = [1 2; 3 4]
+
+# We can do linear algebra:
+
+x = A \ b
+
+#-
+
+A * x
+
+#-
+
+A * x == b
+
+# Note that when multiplying vectors and matrices, dimensions matter. For
+# example, you can't multiply a vector by a vector:
+
+try  #hide
+b * b
+catch err; showerror(stderr, err); end  #hide

```

Comment:
I had to use a hack to capture errors in the output. Here's the upstream https://github.com/JuliaDocs/Documenter.jl/issues/1420#issuecomment-770539595